src/Benchmarks.ClientJob/Benchmarks.ClientJob.csproj

```diff
@@ -4,12 +4,12 @@
 
   <PropertyGroup>
     <Description>Class to transfer data to/from the benchmark client.</Description>
-    <TargetFramework>netstandard1.0</TargetFramework>
```

Comment:
I think it depends on where this is used. But, is there any reason *not* to use NCA2.0 here?